docs/Using Price Feeds/l2-sequencer-feeds/l2-sequencer-flag.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: "L2 Sequencer Uptime Feeds"
+permalink: "docs/l2-sequencer-flag/"
+---
+
+Optimistic rollup protocols move all execution off the layer 1 (L1) Ethereum chain, complete execution on a layer 2 (L2) chain, and return the results of the L2 execution back to the L1. These protocols have a [sequencer](https://community.optimism.io/docs/how-optimism-works/#block-production) that executes and rolls up the L2 transactions by batching multiple transactions into a single transaction.
+
+If a sequencer becomes unavailable, it is impossible to access read/write APIs that consumers are using and applications on the L2 network will be down for most users without interacting directly through the L1 optimistic rollup contracts. The L2 has not stopped, but it would be unfair to continue providing service on your applications when only a few users can use them.
+
+To help your applications identify when the sequencer is unavailable, you can use a data feed that tracks the last known status of the sequencer at a given point in time. This is to allow customers to prevent mass liquidations by providing a grace period to allow customers to react to such an event.
+
+Because not all L2 networks are architected the same way, the Arbitrum sequencer feed behaves slightly differently than the Optimism and Metis sequencer feeds. The architecutre and process for handling outages is different. Follow the guide for your specific L2 network.
```

Comment:
```suggestion
Because not all L2 networks are architected the same way, the Arbitrum sequencer feed behaves slightly differently than the Optimism and Metis sequencer feeds. The architecutre and process for handling outages is different. Follow this guide for your specific L2 network.
```

Reply:
This is correct but perhaps I can re-word it to be more clear. I changed it to `Follow the instructions for your specific network; either [Arbitrum](#arbitrum) or [Optimism and Metis](#optimism-and-metis).`